Upstream: No
Reason: Docker by default doesn't enable IPv6, so tests using this will fail on
CI and infra.

diff --git a/libuhttpmock/uhm-server.c b/libuhttpmock/uhm-server.c
index c28f30a..f1822f6 100644
--- a/libuhttpmock/uhm-server.c
+++ b/libuhttpmock/uhm-server.c
@@ -1393,8 +1393,7 @@ uhm_server_run (UhmServer *self)
 	g_main_context_push_thread_default (priv->server_context);
 
 	priv->server_main_loop = g_main_loop_new (priv->server_context, FALSE);
-	soup_server_listen_local (priv->server, 0, (priv->tls_certificate != NULL) ? SOUP_SERVER_LISTEN_HTTPS : 0,
-	                          &error);
+	soup_server_listen_local (priv->server, 0, (priv->tls_certificate != NULL) ? SOUP_SERVER_LISTEN_HTTPS | SOUP_SERVER_LISTEN_IPV4_ONLY :0, &error);
 	g_assert_no_error (error);  /* binding to localhost should never really fail */
 
 	g_main_context_pop_thread_default (priv->server_context);
